homeassistant/components/wmspro/config_flow.py

```diff
@@ -84,6 +84,11 @@ async def async_step_user(
                 if not pong:
                     errors["base"] = "cannot_connect"
                 else:
+                    await hub.refresh()
+                    rooms = set(hub.rooms.keys())
+                    for entry in self.hass.config_entries.async_entries(DOMAIN):
+                        if set(entry.runtime_data.rooms.keys()) == rooms:
```

Comment:
What is the integration hasn't been set up yet (for whatever reason), runtime_data wouldn't exists or have the expected data at that point?

Reply:
Pushed a change to test for that situation and then skip the uninitialized config entry.